JAVA/Pyramid1.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Scanner;
+
+public class Pyramid{
+	public static void main(String[] args) {
+		int row,col;
+		int half;
+
+		while(true) {
+			System.out.println("출력하고 싶은 피라미드의 줄 수를 입력하시오.");
+
+			Scanner sc = new Scanner(System.in);
+			int line = sc.nextInt();
+			half = line;
+
+			if (line>=1) {
```

Comment:
한 뎁스를 줄이기 위해서 아래처럼도 짤 수 있을 것 같아요
```suggestion
			if (line > 1) {
				System.out.println(" 1 이상의 양수를 입력하시오.\n");
				continue;
			}
```